.env.example

```
@@ -0,0 +1,39 @@
+# ============================================================
+# CamPost 환경변수 — 이 파일을 복사해 .env 로 사용
+# cp .env.example .env   (Windows: copy .env.example .env)
+# ============================================================
+
+# ── PostgreSQL ───────────────────────────────────────────────
+POSTGRES_DB=campost
+POSTGRES_USER=campost
+POSTGRES_PASSWORD=campost1234
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The POSTGRES_PASSWORD key should go before the POSTGRES_USER key

(UnorderedKey)